src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_TravelAidScreen.tsx

```diff
@@ -52,9 +56,17 @@ export const Profile_TravelAidScreen = () => {
           <ToggleSectionItem
             text={travelAidToggleTitle}
             value={toggleValue}
-            onValueChange={(checked) =>
-              setPreference({journeyAidEnabled: checked})
-            }
+            onValueChange={(checked) => {
+              analytics.logEvent(
+                'Journey aid',
+                'Journey aid preference toggled',
+                {
+                  enabled: checked,
+                  isScreenReaderEnabled,
```

Comment:
You don't need to add `screenreader: isScreenReaderEnabled` to have a parameter name on the event?

Reply:
This is shorthand for `isScreenReaderEnabled: isScreenReaderEnabled` in javascript. But I did want to consistently use `screenReaderEnabled` here, so thanks for making me notice 😄

Reply:
So from now, it should be `screenReaderEnabled: true/false` in posthog